### PR TITLE
Fixes for Legrand 067775/741811

### PR DESCRIPTION
**Pull request by EXP-Carthage**
* Renaming of functions and variables to be more explicit.
  * legrand_settingAlwaysEnableLed > legrand_settingEnableLedInDark
  * led_when_off > led_in_dark
  * led_when_on > led_if_on
* Add the possibility to activate LED when the device is on.

### CODE REVIEW

File: devices/legrand.js [hunk 1]
```diff
@@ -175,16 +175,16 @@ module.exports = [
...
             tz.legrand_deviceMode, tz.legrand_identify, tz.ballast_config],
         exposes: [e.light_brightness(),
             exposes.numeric('ballast_minimum_level', ea.ALL).withValueMin(1).withValueMax(254)
                 .withDescription('Specifies the minimum brightness value'),
             exposes.numeric('ballast_maximum_level', ea.ALL).withValueMin(1).withValueMax(254)
                 .withDescription('Specifies the maximum brightness value'),
             exposes.binary('device_mode', ea.ALL, 'dimmer_on', 'dimmer_off').withDescription('Allow the device to change brightness'),
-            exposes.binary('led_when_off', ea.ALL, 'ON', 'OFF').withDescription('Enable the LED when the light is off'),
-            exposes.binary('led_when_on', ea.ALL, 'ON', 'OFF').withDescription('Enables the LED when the light is on')],
+            exposes.binary('led_in_dark', ea.ALL, 'ON', 'OFF').withDescription('Enable the LED when the light is off'),
```
Comment (Koenkk) [inline devices/legrand.js:186]: Please also update the description (also for `led_if_on`)

Comment (EXP-Carthage) [reply devices/legrand.js]: It's done !

File: devices/legrand.js [hunk 2]
```diff
@@ -344,11 +345,11 @@ module.exports = [
         vendor: 'Legrand',
         description: 'Double wired switch with neutral',
         fromZigbee: [fz.on_off, fz.legrand_binary_input_on_off, fz.legrand_cluster_fc01],
-        toZigbee: [tz.on_off, tz.legrand_settingAlwaysEnableLed, tz.legrand_settingEnableLedIfOn],
+        toZigbee: [tz.on_off, tz.legrand_settingEnableLedInDark, tz.legrand_settingEnableLedIfOn],
         exposes: [e.switch().withEndpoint('left'),
             e.switch().withEndpoint('right'),
-            exposes.binary('led_when_off', ea.ALL, 'ON', 'OFF').withDescription('Enable the LED when the light is off'),
-            exposes.binary('led_when_on', ea.ALL, 'ON', 'OFF').withDescription('Enables the LED when the light is on')],
+            exposes.binary('led_in_dark', ea.ALL, 'ON', 'OFF').withDescription('Enables the LED when the light is turned off'),
```
Comment (Koenkk) [inline devices/legrand.js:351]: I think it make sense to also mention `dark` in the description somewhere, given this description `led_if_off` makes more sense.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**